tiatoolbox/tools/stainextract.py

```diff
@@ -241,7 +246,8 @@ class VahadaneExtractor:
 
     """
 
-    def __init__(self, luminosity_threshold=0.8, regularizer=0.1):
+    def __init__(self, luminosity_threshold=0.8, regularizer=0.1) -> None:
+        """Initializes :class:`VahadaneExtractor`."""
```

Comment:
```suggestion
        """Initialize :class:`VahadaneExtractor`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090